src/ui/ozone/platform/wayland/host/wayland_connection.cc

```diff
@@ -391,8 +391,8 @@ void WaylandConnection::Global(void* data,
       return;
     }
     wl_seat_add_listener(connection->seat_.get(), &seat_listener, connection);
-    connection->CreateDataObjectsIfReady();
 #endif  // !defined(USE_NEVA_APPRUNTIME)
+    connection->CreateDataObjectsIfReady();
```

Comment:
Okay sounds reasonable, but why not leaving it there as well? Shouldn't also be enabled for !defined(NEVA_RUNTIME)

Reply:
That's exactly what it is doing. Instead of only be in one of the #if defined branches, it is right after the conditional code so it happens in both cases.

Reply:
Hmmm, :eyeglasses: that's correct, it actually is for both cases, albeit got a bit distracted by the ifdefs.